services/repository/fork.go

```diff
@@ -52,6 +52,7 @@ type ForkRepoOptions struct {
 	Name         string
 	Description  string
 	SingleBranch string
+	Reparent     bool
```

Comment:
I prefer the previous implementation to have a new API to do the reparent.

Reply:
If we do a new API call here, can we still use the `repo_service.ForkRepoOptions{Reparent:true}` or split off that logic on top of a fork?

Reply:
Yes, I think introducing a new API endpoint would be more appropriate. Also, the Reparent field in ForkRepoOptions feels a bit awkward.